eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version4/ChannelCodecs4.scala

```diff
@@ -163,7 +164,9 @@ private[channel] object ChannelCodecs4 {
       .typecase(0x12, claimLocalAnchorOutputTxCodec)
       .typecase(0x11, claimLocalAnchorOutputTxBlockHeightConfirmCodec)
       .typecase(0x01, claimLocalAnchorOutputTxNoConfirmCodec)
-      .typecase(0x02, claimRemoteAnchorOutputTxCodec)
+      .typecase(0x02, unusedRemoteAnchorOutputTxCodec)
+
+    def filterUnusedRemoteAnchorOutputTx(anchorTxs: List[ClaimAnchorOutputTx]): List[ClaimAnchorOutputTx] = anchorTxs.filter(_.confirmationTarget != ConfirmationTarget.Absolute(BlockHeight(42)))
```

Comment:
this is a bit strange but 42 is an impossible block height  and I don't see how we can do better.